src/EditorFeatures/Core/Copilot/RoslynProposalAdjusterProvider.cs

```diff
@@ -133,7 +147,12 @@ bool ReportFailureTelemetry(Exception ex)
         }
     }
 
-    private async Task<(ProposalBase newProposal, bool adjustmentsProposed)> AdjustProposalAsync(
+    /// <summary>
+    /// Returns <see langword="null"/> if we didn't make any adjustments to the proposal.  Otherwise, returns the attempted
+    /// adjusted proposal based on the edits we tried to make.  Note that this does not guarantee that the edits were successfully
+    /// applied to the original edits.  The <see cref="Proposal"/> system may reject them based on their own criteria.
+    /// </summary>
+    private async Task<ProposalBase?> AdjustProposalAsync(
         Solution solution, ProposalBase proposal, CancellationToken cancellationToken)
```

Comment:
this is just cleanup. i prefered just returning a result and having the boolean values computed at the callsite.